Field/index.jsx

```diff
@@ -225,7 +225,7 @@ const Field = React.createClass({
           borderColor: this.state.hover || focus
             ? customize.borderColorSelected
             : customize.borderColor,
-          boxShadow: focus && `0 0 4px ${customize.borderColorSelected}`
+          boxShadow: focus && (customize.boxShadow ? customize.boxShadow : `0 0 4px ${customize.borderColorSelected}`)
```

Comment:
Where is `customize.boxShadow` coming from? In the `withTheme` HoC is not being set